File: devpractice/dotnet/oops&collections/EmployeeManagement/Manager.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EmployeeManagement
{
    public class Manager : Employee
    {
        public string _team;

        public Manager(string team, string name, int empId) : base(name, empId)
        {
            _team = team;
        }

        public override void workToDo()
        {
            Console.WriteLine($"Manager manages team {_team}");
        }
    }
}

File: devpractice/java/employee_management/src/main/java/org/example/Manager.java
package org.example;

public class Manager extends Employee{
    private String team;

    public Manager(String team, String name, int empId) {
        super(name, empId);
        this.team = team;
    }


    @Override
    public void work() {
        System.out.println(this.name + " manages team "+this.team);
    }
}
